lzy/allocator/src/main/java/ai/lzy/allocator/alloc/impl/kuber/NetworkPolicyManager.java

```diff
@@ -0,0 +1,11 @@
+package ai.lzy.allocator.alloc.impl.kuber;
+
+import java.util.List;
+
+public interface NetworkPolicyManager {
+    void createNetworkPolicy(String sessionId, List<PolicyRule> whitelistCIDRs);
+
+    void deleteNetworkPolicy(String sessionId);
+
+    record PolicyRule(String cidr, List<Integer> ports) {}
```

Comment:
Why we need this custom object?
We can use just proto message

Reply:
I think that using classes or records is better that using generated proto-classes, because they're basically DTOs and not very suitable for internal use.